src/txn/txn_rollback_to_stable.c

```diff
@@ -124,6 +124,27 @@ __rollback_abort_update(WT_SESSION_IMPL *session, WT_ITEM *key, WT_UPDATE *first
      * store, it will have a different stop time point.
      */
```

Comment:
The comment just before `if (stable_upd != NULL) {` should be removed as it got moved in the cherry picked commit: https://github.com/wiredtiger/wiredtiger/commit/522d5750c69a95ec791aec244c7891aac105de8e

Reply:
Oops! Thank you for catching this!